app/views/user_mailer/send_email.html.erb

```diff
@@ -5,7 +5,7 @@
   </head>
   <body>
     <h1>Sorry to inform you, <%= @recipient.name %></h1>
-    <p><%= @user.name %> has died :( </p>
+    <p><%= @user.name %> has passed away. <%= @user.name %> cared for you very much, and wanted you to be notified of this unfortunate circumstance.</p>
```

Comment:
Can this say something like "user.name has passed away. user.name cared for you very much and wanted to show you how they would like to be remembered"